src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/xlf/VSPackage.zh-Hant.xlf

```diff
@@ -1,6 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
-  <file datatype="xml" source-language="en" target-language="zh-Hant" original="../VSPackage.resx">
+  <file datatype="xml" source-language="en" target-language="zh-HANT" original="../VSPackage.resx">
```

Comment:
Is this change ok?

Reply:
This wasn't expected, we'll fix it.

Reply:
FYI there are other target language changes in the PR too.

Reply:
The casing shouldn't matter on our end.